src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,29 @@
+package codeu.chat.common;
+
+import codeu.chat.util.Uuid;
+import codeu.chat.util.Time;
+
+public final class ServerInfo {
+  private final static String SERVER_VERSION = "1.0.0";
+
+  public final Time startTime;
+
+  public Uuid version;
+
+  public ServerInfo() {
+
```

Comment:
Super-nit: inconsistent line breaks, other methods have no line break between definition and content